### PR TITLE
Dependency bumps, note especially removes version requirement for web…

### PR DESCRIPTION
**Pull request by cdmo**
…packer

These were all upgraded:

```
➜  myaccount git:(update-dependencies) ✗ yarn outdated
yarn outdated v1.22.4
info Color legend :
 "<red>"    : Major Update backward-incompatible updates
 "<yellow>" : Minor Update backward-compatible features
 "<green>"  : Patch Update backward-compatible bug fixes
Package              Current Wanted Latest Package Type    URL
babel-jest           24.9.0  24.9.0 26.0.1 devDependencies https://github.com/facebook/jest#readme
eslint               6.8.0   6.8.0  7.2.0  devDependencies https://eslint.org
eslint-plugin-node   10.0.0  10.0.0 11.1.0 devDependencies https://github.com/mysticatea/eslint-plugin-node#readme
jest                 24.9.0  24.9.0 26.0.1 devDependencies https://jestjs.io/
typeface-roboto-slab 0.0.72  0.0.72 1.1.4  dependencies    https://github.com/KyleAMathews/typefaces/tree/master/packages/roboto-slab
```

```
warning babel-preset-es2015@6.24.1: 🙌  Thanks for using Babel: we recommend using babel-preset-env now: please read https://babeljs.io/env to update!
warning babel-preset-es2015 > babel-plugin-transform-es2015-block-scoped-functions > babel-runtime > core-js@2.6.11: core-js@<3 is no longer maintained and not recommended for usage due to the number of issues. Please, upgrade your dependencies to the actual version of core-js@3.
```

So I replaced `babel-preset-es2015` with `@babel/preset-env` (see note at https://babeljs.io/docs/en/babel-preset-es2015/)

### CODE REVIEW

**Comment by cdmo** [comment]
Did not know about the new `tag` thing. `content_tag` is getting deprecated :shrug:

https://api.rubyonrails.org/v5.1.7/classes/ActionView/Helpers/TagHelper.html#method-i-tag

**Comment by cdmo** [comment]
also ignroe circle ci failing, it is being weird, I pinged @whereismyjetpack about it

**Comment by cdmo** [comment]
Ah might be CC upgrade. Don’t merge yet

**Comment by cdmo** [comment]
ok yup, good to go now

File: app/components/dashboard_item_component.rb [hunk 1]
```diff
@@ -13,10 +13,10 @@ def card_body
     return nil if @items.nil?
 
     facts = @items.map do |item|
-      content_tag :li, "#{item[:count]}  #{item[:label]}"
+      tag.li "#{item[:count]}  #{item[:label]}"
```
Comment (banukutlu) [inline app/components/dashboard_item_component.rb:16]: aha wow I did not know either, thanks for updating!

**Pull request merged**